### PR TITLE
Upgrade to latest Sync 1.9.1 / Core 2.8.0

### PR DESCRIPTION
**Pull request by cmelchior**

### CODE REVIEW

**Comment by cmelchior** [comment]
Ready for review @realm/java

File: dependencies.list [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Realm Sync Core release used by Realm Java
 # https://github.com/realm/realm-sync/releases
-REALM_SYNC_VERSION=1.8.5
-REALM_SYNC_SHA256=71e70f83b1604672bbfd7c04fcf984ecc8ac683864943ef3de262994c2e7b7fc
+REALM_SYNC_VERSION=1.9.1
```
Comment (nhachicha) [inline dependencies.list]: OS commit below reference a dependency to `1.9.0`

File: dependencies.list [hunk 2]
```diff
@@ -10,4 +10,4 @@ REALM_SYNC_SHA256=71e70f83b1604672bbfd7c04fcf984ecc8ac683864943ef3de262994c2e7b7
 # /tools/sync_test_server/Dockerfile specify which repo (apt) we should
 # install/use between 'realm' and 'realm-testing', the version below should
 # correspond to an existing version on the *specified* repo.
-REALM_OBJECT_SERVER_DE_VERSION=1.6.0-35
+REALM_OBJECT_SERVER_DE_VERSION=1.7.5-180
```
Comment (nhachicha) [inline dependencies.list]: carriage return removed

File: dependencies.list (hunk 1 above)
Comment (nhachicha) [inline dependencies.list]: OS commit below reference the `1.9.0`

File: tools/sync_test_server/Dockerfile [hunk 3]
```diff
@@ -5,8 +5,8 @@ ARG ROS_DE_VERSION
 # Add realm repo
 RUN apt-get update -qq \
     && apt-get install -y curl npm \
-    # && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh \
-    && curl -s https://packagecloud.io/install/repositories/realm/realm-testing/script.deb.sh | bash
+    && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh | bash
```
Comment (nhachicha) [inline tools/sync_test_server/Dockerfile]: were you testing with stable build of ROS?

**Comment by beeender** [comment]
Oh, it should go to master since it is not compatible with previous ROS. So 👍

File: tools/sync_test_server/Dockerfile (hunk 3 above)
Comment (cmelchior) [inline tools/sync_test_server/Dockerfile]: Yes. Testing releases are not getting correctly deployed, so the stable release is the latest available one.

File: dependencies.list (hunk 1 above)
Comment (cmelchior) [inline dependencies.list]: I bumped the commit to the latest possible. It seems that OS was never built against 1.9.1, but there is no breaking changes so It should be okay.

**Pull request merged**